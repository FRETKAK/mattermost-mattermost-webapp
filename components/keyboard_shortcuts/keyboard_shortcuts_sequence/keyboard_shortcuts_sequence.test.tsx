// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';

import {mountWithIntl} from 'tests/helpers/intl-test-helper';

import KeyboardShortcutsSequence from './keyboard_shortcuts_sequence';

describe('components/shortcuts/KeyboardShortcutsSequence', () => {
    test('should match snapshot modal for Mac', () => {
        const wrapper = mountWithIntl(
            <KeyboardShortcutsSequence
                shortcut={{
                    id: 'test',
                    defaultMessage: 'Keyboard Shortcuts\tâŒ˜|/',
                }}
            />,
        );

        expect(wrapper).toMatchSnapshot();
    });
});
