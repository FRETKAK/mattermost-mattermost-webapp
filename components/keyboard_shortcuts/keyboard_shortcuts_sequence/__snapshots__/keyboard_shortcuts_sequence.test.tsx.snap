// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/shortcuts/KeyboardShortcutsSequence should match snapshot modal for Mac 1`] = `
<Memo(KeyboardShortcutSequence)
  shortcut={
    Object {
      "defaultMessage": "Keyboard Shortcuts	⌘|/",
      "id": "test",
    }
  }
>
  <div
    className="shortcut-line"
  >
    <span>
      Keyboard Shortcuts
    </span>
    <ShortcutKey
      key="⌘"
      variant="shortcut"
    >
      <mark
        className="shortcut-key shortcut-key--shortcut-modal"
      >
        ⌘
      </mark>
    </ShortcutKey>
    <ShortcutKey
      key="/"
      variant="shortcut"
    >
      <mark
        className="shortcut-key shortcut-key--shortcut-modal"
      >
        /
      </mark>
    </ShortcutKey>
  </div>
</Memo(KeyboardShortcutSequence)>
`;
